google/cloud/bigtable/internal/table_test.cc

```diff
@@ -67,6 +67,19 @@ struct MockRpcFactory {
 
 }  // anonymous namespace
 
+TEST_F(NoexTableTest, ChangeOnePolicy) {
+  bigtable::noex::Table table(client_, "some-table",
+                              bigtable::AlwaysRetryMutationPolicy());
+  EXPECT_THAT(table.table_name(), ::testing::HasSubstr("some-table"));
```

Comment:
This doesn't really test anything about the policy except that it compiles, right? Should this do a retry on a few (mock) errors to demonstrate the retry policy, or is that overkill?

Reply:
Correct, there other test that actually use the policy, for example:

https://github.com/GoogleCloudPlatform/google-cloud-cpp/blob/f6fa75d463c522072f46e5ef3cfe5918efc31fdd/google/cloud/bigtable/table_bulk_apply_test.cc#L187

I *could* create an accessor to read the policy back and figure out if it is of the desired type, but that feels icky.  Thoughts?

Reply:
No worries, as long as there's an existing test for this, SGTM.